Project.toml

```diff
@@ -16,6 +16,7 @@ ModelingToolkit = "961ee093-0014-501f-94e3-6117800e7a78"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+RuntimeGeneratedFunctions = "7e49a35a-f44a-4d26-94aa-eba1b4ca6b47"
```

Comment:
This needs a compat entry too.

Reply:
Should I just put `= "0.5.12"`?

Reply:
MTK uses RuntimeGeneratedFunctions = "0.5.9" so maybe go with that?